src/Console/Command/Init.php

```diff
@@ -44,42 +53,49 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             ],
             'Web application',
         );
+
         $templateDirectory = [
             'Web application' => 'http',
             'Event-driven function' => 'function',
         ][$choice];
 
-        $fs = new Filesystem;
-        $rootPath = dirname(__DIR__, 3) . "/template/$templateDirectory";
+        $this->rootPath = dirname(__DIR__, 3) . "/template/$templateDirectory";
+
+        self::createFile('index.php');
+        self::createFile('serverless.yml');
 
-        $io->writeln('Creating index.php');
-        $fs->copy("$rootPath/index.php", 'index.php');
+        $io->success('Project initialized and ready to test or deploy.');
 
-        $io->writeln('Creating serverless.yml');
+        return 0;
+    }
 
-        $template = file_get_contents("$rootPath/serverless.yml");
+    /**
+     * Creates files from the template directory and automatically adds
+     * them to git
+     */
+    private function createFile(string $file): void
+    {
+        $overwrite = true;
 
-        $template = str_replace('PHP_VERSION', PHP_MAJOR_VERSION . PHP_MINOR_VERSION, $template);
+        $this->io->writeln("Creating $file");
 
-        file_put_contents('serverless.yml', $template);
+        if (file_exists($file)) {
+            $overwrite = $this->io->confirm("An $file file already exists, do you want to overwrite it?", true);
```

Comment:
`An` should be followed by a vowel and that's not the case

May be we shoudl phrase it like this ?

```suggestion
            $overwrite = $this->io->confirm("A file named $file already exists, do you want to overwrite it?", true);
```